core/activity/src/common.rs

```diff
@@ -73,8 +73,14 @@ pub(crate) fn generate_id() -> String {
     Uuid::new_v4().to_simple().to_string()
 }
 
-pub(crate) async fn _get_activities(db: &DbExecutor) -> Result<Vec<String>, Error> {
-    Ok(db.as_dao::<ActivityDao>()._get_activity_ids().await?)
+pub(crate) async fn get_activities(
+    db: &DbExecutor,
+    max_items: Option<u32>,
```

Comment:
It's not good idea to have `max_items` without `after_timestamp`. In practice it is unusable, because have no way to get older items when you exceeded `max_items`.